docs/content/users/quickstart.md

```diff
@@ -130,14 +130,16 @@ ddev launch
 
 ## Drupal
 
-=== "Drupal 10"
+=== "Drupal 8+"
 
     ```bash
-    mkdir my-drupal10-site
-    cd my-drupal10-site
-    ddev config --project-type=drupal10 --docroot=web
+    mkdir my-drupal-site
+    cd my-drupal-site
+    ddev config --project-type=drupal --docroot=web
     ddev start
-    ddev composer create drupal/recommended-project
+    ddev composer create drupal/recommended-project:^10
+    # For Drupal9: ddev composer create drupal/recommended-project:^9
```

Comment:
In line with my previous comment i would simply drop the how to install drupal 9 and only cover versions that are officially supported here? in the latest docs there was only drupal 10 in this tab. which got already extended by the dev version for drupal 11.x-dev

Reply:
drupal9 is still widely talked about and used, no reason not to leave it in there for a while. Let's remove it in a few months or a year. It also gives a good demonstration of how to use explicit composer recipe pattern.